dotnet/src/Functions/Functions.OpenApi/Model/RestApiSpecification.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// The REST API specification.
+/// </summary>
+public sealed class RestApiSpecification
```

Comment:
We can restrict the public usage of this class and RestApiInfo by marking them as internal types.

Reply:
@SergeyMenshykh all of the other model classes are public sealed. Should they all be internal?

Reply:
Ideally, they should be internal as well because they use the parser and the runner that are internal.

Reply:
Created this task: https://github.com/microsoft/semantic-kernel/issues/7281